homeassistant/helpers/script.py

```diff
@@ -494,8 +494,14 @@ def _handle_exception(
         ):
             raise exception
 
-        # Only Home Assistant errors can be ignored.
-        if not isinstance(exception, exceptions.HomeAssistantError):
+        # Only Home Assistant errors and Connection error can be ignored.
+        if not isinstance(
+            exception,
+            (
+                exceptions.HomeAssistantError,
+                exceptions.ConnectionError,
```

Comment:
This should not be managed here, an integration needs to handle other exceptions/errors and throw an `HomeAssistantError` in case of an issue.

../Frenck